spec/features/variables/variable_with_draft_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+describe 'Variable with draft' do
+  modal_text = 'requires you change your provider context to MMT_2'
+
+  context 'when viewing a variable that has an open draft' do
+    before do
+      login
+    end
+
+    context 'when the variables provider is the users current provider', js: true do
+      before do
+        ingest_response, @concept_response = publish_variable_draft(include_new_draft: true)
+
+        visit variable_path(ingest_response['concept-id'])
+      end
+
+      it 'displays a message that a draft exists' do
+        expect(page).to have_content('This variable has an open draft associated with it. Click here to view it.')
+      end
+
+      context 'when clicking the link' do
+        before do
+          click_on 'Click here to view it.'
+        end
+
+        it 'displays the draft' do
+          within '.eui-breadcrumbs' do
+            expect(page).to have_content('Drafts')
```

Comment:
not totally necessary, but you could make this more specific and check for 'Variable Drafts'